docs/reference/cli.md

```diff
@@ -9,46 +9,32 @@ title: CLI Reference
 
 Login to Infra
 
-#### Description
-
-Login to Infra and start a background agent to keep local configuration up-to-date
-
 ```bash
-infra login [SERVER] [flags]
+infra login SERVER [flags]
 ```
 
 #### Examples
 
 ```bash
-# By default, login will prompt for all required information.
-$ infra login
 
-# Login to a specific server
-$ infra login infraexampleserver.com
+# Login
+$ infra login my.infrahq.com
 
-# Login with a specific identity provider
-$ infra login --provider okta
-
-# Login with an access key
-$ export INFRA_ACCESS_KEY=1M4CWy9wF5.fAKeKEy5sMLH9ZZzAur0ZIjy
-$ infra login
-
-# Login with pre-set provider and server
-$ export INFRA_SERVER=example.infrahq.com
-$ export INFRA_PROVIDER=google
-$ infra login
+# Login with username (will prompt for password)
+$ infra login --user me@example.com
+		
 ```
 
 #### Options
 
 ```console
-      --key string                       Login with an access key
       --no-agent                         Skip starting the Infra agent in the background
       --non-interactive                  Disable all prompts for input
-      --provider string                  Login with an identity provider
+  -p, --password string                  Password
```

Comment:
I think generally it's considered a bad idea to specify the password on the command line, because it can be see by anyone with access to `/proc`, and it also ends up in files like the bash history file (and probably others) that may be world readable.

For non-interactive the environment variable works well. That's still visible in `/proc`, but at least won't end up in log files in all cases.

`psql` and probably others have a `--password` flag to force a prompt. I think that'd be a nice way to do this. 
1. If someone sets `--user <username>` they only get prompted for password
2. If someone sets `--password` they get prompted for both
3. If they set both it's no different than `--user <username>`

I'd also suggest we remove `-p`. We don't need short flags for most operations, and it in the future we might want it for something else.

```suggestion
      --password string                  Prompt for a password.
```

Reply:
I opted to just remove the `--password` flag, as it's the same as providing `--user`.